anthos-onprem-terraform/avmw_user_cluster_metallb/README.md

```diff
@@ -50,3 +50,25 @@ have changed directory to where this samples is:
 
     You can view your user cluster in the
     [Anthos clusters page](https://console.cloud.google.com/anthos/clusters).
+
+### Upgrade the user cluster with terraform
+
+Before upgrading the user cluster, please make sure the admin cluster of the target version has been upgraded. The steps to upgrade the admin cluster is listed in the [public documentation](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrading#:~:text=In%20the%20Cluster%20basics%20section,Click%20Upgrade.).
```

Comment:
Unlike ABM, upgrading the AVMW admin cluster first isn't a requirement for AVMW. What I will be working on for the public doc is adding  a Terraform tab to this doc: https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrade-on-prem-api and include the `gkectl prepare --upgrade-platform` step since upgrades via Terraform require this.

Reply:
Updated accordingly by guiding users to upgrade the platform controller using this command instead of the admin cluster. Thanks for correction!